dev-cpp/notcurses/notcurses-2.2.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 nick black
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/notcurses/archive/v2.2.1.tar.gz"
```

Comment:
Please replace this with
```
SRC_URI="https://github.com/dankamongmen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Reply:
Done, thanks. FWIW, the packaging guide recommends against this usage (https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds).